troubleshooting/ETW/Readme.md

```diff
@@ -116,6 +116,77 @@ StatusMonitor uses TraceEventSession to record ETW logs.
 - https://github.com/dotnet/roslyn/wiki/Recording-performance-traces-with-PerfView
 - https://github.com/microsoft/perfview/blob/master/src/TraceEvent/TraceEventSession.cs
 
+### Self Diagnostics
+
+This SDK also ships a "self-diagnostics feature", which helps troubleshooting.
```

Comment:
I think you should mention here that "as of 2.18.0, this sdk contains....".
We wouldn't want users of older version to get stuck trying to use this.